commands/FBDebugCommands.py

```diff
@@ -423,9 +423,9 @@ def isHeap(addr):
 
     allocations = (addr for addr in pointers if isHeap(addr))
     for addr in allocations:
-        print >>self.result, '0x{addr:x} {path}'.format(addr=addr, path=pointers[addr])
```

Comment:
I need some advice on this.

Reply:
This might help.

https://stackoverflow.com/questions/34385674/what-does-print-do-in-python

I suspect this means we should write it as

`print('0x{addr:x} {path}'.format(addr=addr, path=pointers[addr], file=self.result)`

Reply:
Yep